js/color-view.js

```diff
@@ -0,0 +1,56 @@
+import { html, render } from 'https://unpkg.com/lit-html@1.0.0/lit-html.js';
+
+class ColorPicker extends HTMLElement {
+    constructor() {
+        super();
+        this.render = this.render.bind(this);
+        this.setColor = this.setColor.bind(this);
+        this.applyColor = this.applyColor.bind(this);
+        this.onContextChanged = this.onContextChanged.bind(this);
+
+        fin.me.on('host-context-changed', this.onContextChanged);
+
+        // Set initial color based on current context value
+        fin.Platform.getCurrentSync().getWindowContext().then(initialContext => {
+            if (initialContext && initialContext.color) {
+                this.applyColor(initialContext.color);
+            }
+        });
+        this.render();
+    }
+
+    async render() {
+        const content = html`
+        <fieldset>
+            <p>
+                I am context-sensitive
+            </p>
+            <form @submit=${this.setColor}>
+                <input type="text" placeholder="Enter color" autofocus>
+                <button action="submit">Set Color</button>
+            </form>
+            <button @click=${fin.me.showDeveloperTools}>
```

Comment:
Haha and here it turns out we actually do have to bind when we're using lit (since it's passing the function, rather than eval'ing)
```suggestion
            <button @click=${fin.me.showDeveloperTools.bind(fin.me)}>
```